### PR TITLE
Eliminated issues after demo 23-04-21

### PR DESCRIPTION
**Pull request by TarasOrlovskyi**
1. Changed font in project from Nunito to Nunito with cyrillic (common.css, new
   folder with font)
2. First vinyl image setted 548x548 px
3. Swaped artist & release into vinyl.html, search.html, catalog.html.
   Added three dots if longer into release, artist, caatalog.
4. Deleted catalog number from after search page

### CODE REVIEW

**Review by AlexKievDeveloper — DISMISSED** [review]

File: src/main/resources/templates/vinyl.html [hunk 1]
```diff
@@ -141,7 +147,8 @@
                     </div>
                 </div>
 
-                <p class="content-title" th:if="${not #lists.isEmpty(vinylsByArtist)}" th:text="'More by '+${firstVinyl.artist}">
+                <p class="content-title" th:if="${not #lists.isEmpty(vinylsByArtist)}"
+                   th:text="'More by '+${firstVinyl.artist}">
                     No more release
                 </p>
```
Comment (Greengrer) [inline src/main/resources/templates/vinyl.html:153]: *releases

File: src/main/resources/templates/vinyl.html [hunk 2]
```diff
@@ -152,8 +159,10 @@
                             <a class="other-vinyls__image" th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}">
                                 <img th:src="${vinylByArtist.imageLink}" alt="vinyl image">
                             </a>
-                            <a th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}" th:text="${vinylByArtist.release}" class="other-vinyls__realise">No realise</a>
-                            <a th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}" th:text="${vinylByArtist.artist}" class="other-vinyls__artist">No artist</a>
+                            <a th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}" th:text="${vinylByArtist.artist}"
+                               class="other-vinyls__artist">No artist</a>
+                            <a th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}"
+                               th:text="${vinylByArtist.release}" class="other-vinyls__realise">No realise</a>
                         </div>
```
Comment (Greengrer) [inline src/main/resources/templates/vinyl.html]: release, not realise

File: src/main/resources/templates/vinyl.html [hunk 3]
```diff
@@ -152,8 +159,10 @@
                             <a class="other-vinyls__image" th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}">
                                 <img th:src="${vinylByArtist.imageLink}" alt="vinyl image">
                             </a>
-                            <a th:href="@{oneVinyl(vinylId=${vinylByArtist.vinylId})}" th:text="${vinylByArtist.release}" class="other-vinyls__realise">No realise</a>
```
Comment (Greengrer) [inline src/main/resources/templates/vinyl.html:155]: release, not realise

File: src/main/resources/static/css/pagesWithVinyls.css [hunk 4]
```diff
@@ -37,13 +38,56 @@
 }
 
 .first-vinyl__realise{
+  font-size: 1.34em;
```
Comment (Greengrer) [inline src/main/resources/static/css/pagesWithVinyls.css]: release, not realise

File: src/main/resources/static/css/pagesWithVinyls.css [hunk 5]
```diff
@@ -37,13 +38,56 @@
...
+  top:625px;
+  left: 0;
+}
+
+.vinyl-tip-text {
+  top:685px;
+}
+
+.first-vinyl__realise:hover .tip-text {
+  visibility: visible;
```
Comment (Greengrer) [inline src/main/resources/static/css/pagesWithVinyls.css]: release, not realise

File: src/main/resources/templates/search.html [hunk 6]
```diff
@@ -104,9 +104,8 @@
                   <a class="first-vinyl__image" th:href="@{oneVinyl(vinylId=${firstVinyl.vinylId})}">
                     <img th:src="${firstVinyl.imageLink}" alt="first vinyl">
                   </a>
-                  <a th:text="${firstVinyl.release}" class="first-vinyl__realise" th:href="@{oneVinyl(vinylId=${firstVinyl.vinylId})}">No release</a>
-                  <a th:text="${firstVinyl.artist}" class="first-vinyl__artist" th:href="@{oneVinyl(vinylId=${firstVinyl.vinylId})}">No artist</a>
-                  <div>Catalog No: MFO 42902</div>
+                  <a th:utext="${firstVinyl.artist}+'&lt;span class=&quot;tip-text&quot;&gt;'+${firstVinyl.artist}+'&lt;/span&gt;'" class="first-vinyl__artist" th:href="@{oneVinyl(vinylId=${firstVinyl.vinylId})}">No artist</a>
+                  <a th:utext="${firstVinyl.release}+'&lt;span class=&quot;tip-text&quot;&gt;'+${firstVinyl.release}+'&lt;/span&gt;'" class="first-vinyl__realise" th:href="@{oneVinyl(vinylId=${firstVinyl.vinylId})}">No release</a>
```
Comment (Greengrer) [inline src/main/resources/templates/search.html]: release, not realise

File: src/main/resources/templates/search.html [hunk 7]
```diff
@@ -116,8 +115,8 @@
                 <a class="other-vinyls__image" th:href="@{oneVinyl(vinylId=${vinylFromFirstRow.vinylId})}">
                   <img th:src="${vinylFromFirstRow.imageLink}" alt="vinyl image">
                 </a>
-                <a th:text="${vinylFromFirstRow.release}" th:href="@{oneVinyl(vinylId=${vinylFromFirstRow.vinylId})}" class="other-vinyls__realise">No release</a>
                 <a th:text="${vinylFromFirstRow.artist}" th:href="@{oneVinyl(vinylId=${vinylFromFirstRow.vinylId})}" class="other-vinyls__artist">No artist</a>
+                <a th:text="${vinylFromFirstRow.release}" th:href="@{oneVinyl(vinylId=${vinylFromFirstRow.vinylId})}" class="other-vinyls__realise">No release</a>
               </div>
```
Comment (Greengrer) [inline src/main/resources/templates/search.html]: release, not realise (in class name)

File: src/main/resources/templates/search.html [hunk 8]
```diff
@@ -132,8 +131,8 @@
               <a class="other-vinyls__image" th:href="@{oneVinyl(vinylId=${vinylFromOtherRow.vinylId})}">
                 <img th:src="${vinylFromOtherRow.imageLink}" alt="vinyl image">
               </a>
-              <a th:text="${vinylFromOtherRow.release}" th:href="@{oneVinyl(vinylId=${vinylFromOtherRow.vinylId})}" class="other-vinyls__realise">No release</a>
               <a th:text="${vinylFromOtherRow.artist}" th:href="@{oneVinyl(vinylId=${vinylFromOtherRow.vinylId})}" class="other-vinyls__artist">No artist</a>
+              <a th:text="${vinylFromOtherRow.release}" th:href="@{oneVinyl(vinylId=${vinylFromOtherRow.vinylId})}" class="other-vinyls__realise">No release</a>
```
Comment (Greengrer) [inline src/main/resources/templates/search.html]: release, not realise in class name.

**Review by Greengrer — CHANGES_REQUESTED** [review]
You're cool! I think that where you wrote "realise" you meant "release".
Like 
No Artist
No Release

Fix it if I'm right? You can right click on directory -> replace all in path -> replace realise to release

**Review by AlexKievDeveloper — APPROVED** [review]

**Review by Greengrer — APPROVED** [review]

**Pull request merged**